### PR TITLE
Storybook code snippet for end of call survey

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Storybook code snippet for end of call survey

# Why
Part of end of call survey dod

# How Tested
Tested with storybook

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/EndOfCallSurvey.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+## End of Call Survey
+
+<DetailedBetaBanner version={'1.11.0-beta.2'} />
+
+Azure Communication Services UI Library is adding support for End of Call Survey.
+With this new feature, users can display star surveys and checkbox surveys at the end of the call to gather feedback on call quality/experience.
+Enable this feature to listen to the end users and improve calling experience. Results from end of call survey is also very helpful for Azure Communication Services to diagose potential calling issues and improve overall calling experience.
```
Comment (edwardlee-msft) [inline packages/storybook/stories/EndOfCallSurvey.stories.mdx:17]:
```suggestion
Enable this feature to listen to the end users and improve calling experience. Results from end of call survey is also very helpful for Azure Communication Services to diagnose potential calling issues and improve overall calling experience.
```

File: packages/storybook/stories/EndOfCallSurvey.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+<Stack tokens={{ childrenGap: '3rem' }}>
+  <Stack.Item align="center">
+    <img src="images/enableFreeFormText.png" alt="example of enabling free form text survey and using survey data" />
+  </Stack.Item>
+</Stack>
+
+### Incorporating End of Call Survey into your experience
+
+The UI Library enables users to display surveys at end of call by providing the option to use end of call survey within the CallComposite and CallWithChat experiences.
```
Comment (edwardlee-msft) [inline packages/storybook/stories/EndOfCallSurvey.stories.mdx:37]:
```suggestion
The UI Library enables users to display surveys at end of call by providing the option to use end of call survey within the CallComposite and CallWithChatComposite experience.
```

File: packages/storybook/stories/EndOfCallSurvey.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+End of Call Survey can be enabled both in Mobile Web sessions and in Desktop Web sessions.
+End of Call Survey is enabled by default and are automatically included within the CallComposite and CallWithChatComposite experiences.
```
Comment (edwardlee-msft) [inline packages/storybook/stories/EndOfCallSurvey.stories.mdx:39]:
```suggestion
End of Call Survey is enabled by default and is automatically included within the CallComposite and CallWithChatComposite experience.
```

**Comment by edwardlee-msft** [comment]
For the enableFreeFormText.png, include a comment above the console.log indicating Contoso should add their own logic and that the console.log is a placeholder and should be replaced. 
image

**Comment by dmceachernmsft** [comment]
> For the enableFreeFormText.png, include a comment above the console.log indicating Contoso should add their own logic and that the console.log is a placeholder and should be replaced. image

These images should be code snippets so Contoso can copypaste them

**Comment by edwardlee-msft** [comment]
Donald brings up a good point, are you able to create code snippets rather than screenshots of code?

**Pull request closed without merging**